e2e/pages/enterApplicant/enterApplicant.js

```diff
@@ -3,46 +3,60 @@ const postcodeLookup = require('../../fragments/addressPostcodeLookup');
 
 module.exports = {
 
-  fields: {
-    applicant: {
-      name: '#applicant_name',
-      nameOfPersonToContact: '#applicant_personToContact',
-      jobTitle: '#applicant_jobTitle',
-      address: '#applicant_address_address',
-      mobileNumber: '#applicant_mobile',
-      telephoneNumber: '#applicant_telephone',
-      email: '#applicant_email',
-      pbaNumber: '#applicant_pbaNumber',
-    },
-    solicitor: {
-      name: '#solicitor_name',
-      mobileNumber: '#solicitor_mobile',
-      telephoneNumber: '#solicitor_telephone',
-      email: '#solicitor_email',
-      dx: '#solicitor_dx',
-      reference: '#solicitor_reference',
-    },
+  state: {
+    context: 0,
+  },
+
+  fields: function () {
+    const id = this.state.context;
+
+    return {
+      applicant: {
+        partyType: `#applicants_${id}_party_partyType`,
+        name: `#applicants_${id}_party_name`,
+        address: `#applicants_${id}_party_address_address`,
+        email: `input[id="applicants_${id}_party_emailAddress_email"]`,
+        telephone: `input[id="applicants_${id}_party_telephoneNumber_telephoneNumber"]`,
+        nameOfPersonToContact: `input[id="applicants_${id}_party_telephoneNumber_contactDirection"]`,
+        mobileNumber: `#applicants_${id}_party_mobileNumber_telephoneNumber`,
+        jobTitle: `#applicants_${id}_party_jobTitle`,
+        pbaNumber: `input[id="applicants_${id}_party_pbaNumber"]`,
+        leadApplicant: {
+          yes: `#applicants_${id}_leadApplicantIndicator-Yes`,
+          no: `#applicants_${id}_leadApplicantindictor-No`,
+        },
+      },
+      solicitor: {
```

Comment:
It seems that it should be outside as it is same regardless what id is passed in. Please move it.